boards/esp32-wrover-kit/doc.txt

```diff
@@ -92,7 +93,7 @@ ESP-WROVER-KIT has the following on-board components
 
 The following table shows the default board configuration sorted according to the defined functionality of GPIOs for different hardware options. This configuration can be overridden by \ref esp32_app_spec_conf "application-specific configurations".
 
-In the following table following abbreviations are used:
+The following abbreviations are used in following tables:
```

Comment:
"Abbreviations used in the following tables:"?

Reply:
Probably, you are seeing to less in the diff. In fact, a list of abbreviations that are used in subsequent tables is defined first. Maybe, we could say "These abbreviations are used in subsequent tables:" or something like that to avoid the double "following".

> The following abbreviations are used in following tables:
>
><b>SDC</b> = SD-Card interface is used (module **sdcard_spi** is enabled)
><b>CAM</b> = Camera is plugged in/used
>
>Function            | None   | SDC    | CAM    | SDC+CAM | Remarks  | Configuration
>:-------------------|--------|--------|--------|---------|:---------|----------
>ADC_LINE(0)         | GPIO34 | GPIO34 | -      | -       | CAMERA_D6 | \ref esp32_adc_channels "ADC Channels"

Reply:
> or something like that to avoid the double "following"

Yes, that's what I was trying to refer to